api/src/LinkyLink/appsettings.sample.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Why add this into a `sample` extension vs just including it in the top level `appsettings.json` ?

Reply:
I feel like naming it `sample` makes it clearer what kind of settings one can use with the app.  It just a practice that some language use.  No strong feelings either way and can change it back